tests/Bridge/Symfony/Routing/RouteNameResolverTest.php

```diff
@@ -111,4 +112,25 @@ public function testGetRouteNameForCollectionRoute()
 
         $this->assertSame('some_collection_route', $actual);
     }
+
+    public function testGetCorrectRouteNameForItemRoute()
+    {
+        $routeCollection = new RouteCollection();
+        $routeCollection->add('custom_user_item_route', new Route('/auth/me', [
+            '_api_resource_class' => User::class,
+            '_api_item_operation_name' => 'user_custom_item_operation',
+        ]));
+        $routeCollection->add('user_item_route', new Route('/user/{id}', [
+            '_api_resource_class' => User::class,
+            '_api_item_operation_name' => 'user_item_operation',
+        ]));
```

Comment:
I'm not sure to understand. Adding a custom operation first allows to customize the IRI URL.